lib/plugins/aws/invokeLocal/resolvers/lambda.js

```diff
@@ -0,0 +1,23 @@
+const aws = require('aws-sdk');
+async function resolve(action, parameters, resourceName, resource, getCFResources, serverless, serviceConfig) {
```

Comment:
Probably better format for value resolver could be something as:

```javascript
module.exports = {
  resolveRef: (plugin, resource) => {
    // Resolve Ref, possibly from local template, otherwise remotely via AWS SDK
  },
  resolveAtt: (plugin, resource, parameters) => {
    // Resolve Att, possibly from local template, otherwise remotely via AWS SDK
  }
};
```

Reply:
What's your vision for the plugin parameter?

Reply:
Instance of https://github.com/serverless/serverless/blob/7f9f8084304c705139ad20a2ea4d2f44f6693eb1/lib/plugins/aws/invokeLocal/index.js#L21